CIP-0001/README.md

```diff
@@ -1,185 +1,374 @@
-Motivation     | A clear and short explanation that introduces the reason behind a proposal. Changing an established design, must outline design issues that motivate a rework.
-Specification  | The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations.
-Rationale      | The rationale fleshes out the specification by describing what motivated the design and what led to particular design decisions. It should describe alternate designs considered and related work. The rationale should provide evidence of consensus within the community and discuss important objections or concerns raised during the discussion. It must also explain how the proposal affects the backward compatibility of existing solutions when applicable.
-Path to Active | A reference implementation, observable metrics or anything showing the acceptance of the proposal in the community. Any CIP must have an approved 'Path to Active' before it can move to 'Active'. It acts as a high-level roadmap for the proposal.
+Abstract       | A short (\~200 word) description of the proposed solution and the technical issue being addressed.
+Motivation     | A clear explanation that introduces a proposal's purpose, use cases, and stakeholders. If the CIP changes an established design, it must outline design issues that motivate a rework. For complex proposals, authors must write a [Cardano Problem Statement (CPS) as defined in CIP-9999][CPS] and link to it as the `Motivation`.
+Specification  | The technical specification should describe the proposed improvement in sufficient technical detail. In particular, it should provide enough information that an implementation can be performed solely based on the design outlined in the CIP. A complete and unambiguous design is necessary to facilitate multiple interoperable implementations.
+Rationale      | The rationale fleshes out the specification by describing what motivated the design and what led to particular design decisions. It should describe alternate designs considered and related work. The rationale should provide evidence of consensus within the community and discuss significant objections or concerns raised during the discussion. <br/><br/>It must also explain how the proposal affects the backward compatibility of existing solutions when applicable. If the proposal responds to a [CPS][], the 'Rationale' section should explain how it addresses the CPS and answer any questions that the CPS poses for potential solutions.
+Path to Active | Organised in two sub-sections:<br/><h5>Acceptance Criteria</h5>Describes what are the acceptance criteria whereby a proposal becomes _'Active'_.<br/><h5>Implementation Plan</h5>A plan to meet those criteria. Or `N/A` if not applicable.
 Copyright      | The CIP must be explicitly licensed under acceptable copyright terms ([see below](#licensing)).
 
 <details>
```

Comment:
I also thing the collapsible sections don't work as well as straightforward markup would.  It's clear enough from the header division: while collapsing the text under each header can make it seem like the following text is associated with that header (e.g. the links to the template files are easily missed because they appear under "Licensing").

More of a problem: the collapsed sections _aren't searchable_.  People will be looking for keywords in that collapsed text (e.g. header field names or ecosystem tags) and won't be able to find them unless those sections are expanded.

Reply:
+1 for removing the collapsing sections, prefer the entire document scrolleable